javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,51 @@
+// Future age calculator
+const yearOfBirth = 1983; //  data type number
+let yearFuture = 2070;
+let age = yearFuture - yearOfBirth;
+console.log(`you will be ${age} years old in ${yearFuture}`);
+
+
+// Dog age Calculator
+
+let dogYearOfBirth = 2004;
+let dogYearFuture =  2024;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears = true)
+    {
+        console.log(`your dog will became ${dogYear * 7} dog years old in ${dogYearFuture}`);
+            }
+            else {
+                 console.log(`your dog will became ${dogYear} human years is ${dogYearFuture}`);
+            }
```

Comment:
Please try to format the code properly. For every open `{`, indent the code, and for every `}`, index first, so you get something like this:

```js
if(myValue) {
    console.log('my value was true')
}

```

Reply:
I will do